src/database/migrations/versions/2da0dc469be8_temp_migration.py

```
@@ -7,9 +7,8 @@
 """
 from typing import Sequence, Union
 
-from alembic import op
 import sqlalchemy as sa
-
+from alembic import op
```

This migration shortens token columns from 255 to 64 characters. That can cause data truncation or ALTER TABLE failures if any existing token values exceed 64 characters. Before applying this migration in a populated DB consider:

- Adding a pre-migration check to detect tokens longer than 64 and either abort with a clear error or trim/transform them explicitly (e.g., `op.execute("UPDATE ... SET token = substr(token, 1, 64) WHERE length(token) > 64")`) so the alter succeeds.
- Alternatively, add a safety migration step that fails loudly so you can address long tokens manually.

Also, be explicit about `existing_type` so it matches the actual column type in your DB backend (some backends show `VARCHAR`, others `character varying`); mismatches can make Alembic generate incorrect SQL. Finally, document the rationale for shortening tokens so reviewers/DBAs understand the intent.